runestone/fitb/js/fitb.js

```diff
@@ -241,10 +241,9 @@ export default class FITB extends RunestoneBase {
         }
         if (logFlag) {
             // Sometimes we don't want to log the answer--for example, when timed exam questions are re-loaded
-            let ret = await this.logCurrentAnswer();
+            let data = await this.logCurrentAnswer();
```

Comment:
The first promise resolves to a response object.  You need to await the call to response.json() to get the json object.  I'm not following why you made this change.

Reply:
I guess you moved that to logBookEvent?

Reply:
What we want from logBookEvent is the returned JSON -- it's already checking the response, so I see no need to return the response itself.

Do you want me to look at updating logRunEvent?

Reply:
That would be great.  I definitely think those two should be consistent in what they return.

Reply:
Looking through the code, most calls of `logRunEvent` aren't awaited, meaning the calls aren't guaranteed to complete. Should all these be made using either `await` or `.then`, or do I misunderstand the code?

Reply:
Also, I wanted to do a better job of documenting this new interface / approach. Do you have anything written up on this, or should I create it?

Reply:
For better or for worse, the vast majority of the time we have never cared or checked the results of logging.  Just fire off the logging event and move on without worrying if it was successful or not.

Reply:
I have not written anything up, other than the additional comments I may have added here and there as I was working on it.

Reply:
OK, sounds good. I'll run with that. Thanks!